frontend_v2/src/app/components/publiclists/challengelist/challengelist.component.html

```diff
@@ -3,7 +3,7 @@
   <div *ngIf="routerPublic.url === allChallengesRoutePathCommon" [@.disabled]="true">
     <h4 class="fw-light">All challenges</h4>
     <mat-tab-group>
-      <mat-tab label="Ongoing ({{ ongoingChallenges.length }})">
+      <mat-tab label="Ongoing ({{ ongoingChallenges.length }})" id="ongoing-challenges">
```

Comment:
can we please revert this change as its not required

Reply:
Actually we are getting an error as the id [here](https://github.com/Cloud-CV/EvalAI/blob/39bd56239d2eef5a1b4f16ab0eb44be7b5fbdc07/frontend_v2/src/app/components/publiclists/challengelist/challengelist.component.ts#L234) is not assigned anywhere.